challenge.rb

```diff
@@ -1,26 +1,27 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |x| x.capitalize }
```

Comment:
Check this out...
http://ruby-doc.org/core-1.9.3/Symbol.html#method-i-to_proc

`input.map(&:capitalize)` is a thing that you'll see often. In rails production code, you might see something like...

```
users = User.all
users.map(&:firstname)
=> ['Adam', 'Ben']
```

Though `.pluck` is a better option overall. 
http://apidock.com/rails/ActiveRecord/Calculations/pluck